docker-cloud/cloud-swarm/create-cloud-swarm-aws.md

```diff
@@ -0,0 +1,113 @@
+nodes on your behalf in your cloud services provider account.
+
+If you haven't yet linked Docker Cloud to AWS, follow the steps in [Link Amazon Web Services to Docker Cloud](link-aws-swarm.md). Once it's
+linked, it will show up on the **Swarms -> Create** page as a connected service
+provider.
+
+![](images/aws-creds-cloud.png)
+
+## Create a swarm
+
+1. If necessary, log in to Docker Cloud and switch to Swarm Mode
+
+2. Click **Swarms** in the top navigation, then click **Create**.
+
+    Alternatively, you can select **+ -> Swarm** from the top navigation to get to the same page.
+
+3.  Enter a name for the new swarm.
+
+    Your Docker ID is pre-populated. In the example, our swarm name
+    is "vote-swarm".
+
+    ![](images/aws-create-swarm-1-name.png)
+
+    >**Tip:** For Docker Cloud, use all lower case letters for swarm names. No spaces, capitalized letters, or special characters other than `.`, `_`, or `-` are allowed. AWS does not accept underscores in the name `_`.
+
+4.  Select Amazon Web Services as the service provider, select a channel (`Stable` or `Edge`) from the drop-down menu, and provide an App name.
+
+    In this example, we'll use the `Stable` channel, and use "voting_app" as our application name.
+
+    ![](images/aws-create-swarm-0.png)
+
+5.  Select a **Region** from the drop-down menu.
+
+    > **Tip:** The SSH keys available to you in the next steps are
+    filtered by the region you select here. Make sure that you have
+    appropriate SSH keys available on the region you select.
+
+    Optionally, click **Region Advanced Settings** to configure a
+    [Virtual Private Cloud(VPC)](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Introduction.html) on which to run this swarm.
+
+    ![](images/aws-create-swarm-3-region.png)
+
+    For guidance on setting up a VPC, see [Recommended VPC and subnet setup](https://docs.docker.com/docker-for-aws/faqs/#can-i-use-my-existing-vpc) in the Docker for AWS topics.
+
+6.  Choose how many swarm managers and swarm worker nodes to deploy.
+
+    Here, we create one manager and two worker nodes. (This maps nicely to the [Swarm tutorial setup](/engine/swarm/swarm-tutorial/index.md) and the [voting app sample in Docker Labs](https://github.com/docker/labs/blob/master/beginner/chapters/votingapp.md).)
+
+    ![](images/cloud-create-swarm-4-size.png)
+
+7.  Configure swarm properties.
+
+    ![](images/aws-create-swarm-5-properties.png)
+
+    * Select a public SSH key for Docker Cloud to use to connect to the
+    nodes on AWS. Public keys from the [key pairs you configured on AWS](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) are provided in the drop-down menu. Only keys associated with the
+    Region you selected (in step 5) are shown.
+
+    * Choose whether to provide daily resource cleanup.
+
+    * Enable or disable Cloudwatch for container logging. (It is enabled by default.)
+
+    <font style="color:red;">
+    RYAN, ALBERTO:<br>
```

Comment:
Believe the primary advantage of leaving "daily resource cleanup" enabled is to avoid incurring charges for resources that you are no longer using. (cc: @alberto for confirmation)

Btw, here's the AWS doc on the topic: http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_CleaningUp.html